projects/kubernetes/test-infra/update_helm_chart.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+set -e
+set -o pipefail
+set -x
+
+PATCH=patch
+if [[ "$(uname -s)" == "Darwin" ]]; then
+    PATCH=gpatch
+fi
+
+if ! command -v $PATCH > /dev/null; then
```

Comment:
meh, i dont know about this.  We have deps all over that need to be installed and we dont typically try to install them.  Not saying we dont need to do better, but im not sure this is the best approach.  You decide tho if you want to keep it.